src/protocolsupport/protocol/packet/middleimpl/readable/play/v_pe/LoginPacket.java

```diff
@@ -53,14 +56,8 @@ protected void read0(ByteBuf from) {
 		from.readBoolean(); //broadcast to xbl
 		from.readBoolean(); //commands enabled
 		from.readBoolean(); //needs texture pack
-		int gameRuleCount = VarNumberSerializer.readVarInt(from); //game rules
```

Comment:
Why was this removed?

Reply:
was removed from the PSPE support and always reports 0 now:
https://github.com/ProtocolSupport/ProtocolSupport/commit/cd7cbb61acd1262a7085035b120ad8908996e07d#diff-226b405d338e937694734097e3342101R74

Reply:
While it does report 0, that zero still needs to be read from data.

Reply:
still read it right down here:
https://github.com/ProtocolSupport/ProtocolSupportBungee/pull/69/files/1e7d2237011cf09863d12a04d3dce5ad98e274df#diff-10a180b559675d36c25de12bd641660dR59

Reply:
Yeah I removed it from PSPE because I couldn't get it to work yet. But if we reimplement it in PE we need to do it here too. I think the encoding was right, I possibly had wrong names or so. Anyway if it works we might as well keep it the way it was. Otherwise this is fine for now.

Reply:
looks the game rules are statically typed now?
https://github.com/NukkitX/Nukkit/blob/master/src/main/java/cn/nukkit/level/GameRules.java